k9mail/src/main/java/com/fsck/k9/activity/compose/PgpEnabledErrorDialog.java

```diff
@@ -0,0 +1,71 @@
+package com.fsck.k9.activity.compose;
+
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.DialogInterface.OnClickListener;
+import android.os.Bundle;
+import android.support.annotation.IdRes;
+import android.view.LayoutInflater;
+import android.view.View;
+
+import com.fsck.k9.R;
+import com.fsck.k9.view.HighlightDialogFragment;
+
+
+public class PgpEnabledErrorDialog extends HighlightDialogFragment {
+    private static final String ARG_IS_GOTIT = "is_gotit";
```

Comment:
The name reads super weird to me. We should come up with something more descriptive at some point in the future.

Also not sure if "Got it" is understood by users with only basic English skills. But that, too, is a problem we can solve later.

Reply:
"Got It" is the label that google uses in pretty much all of their apps in informative dialogs like this